src/_utils/item/index.tsx

```diff
@@ -107,20 +122,30 @@ const Item = ({
             <Text
               className={rightBody ? 'kirk-item-title' : null}
               display={rightTitleDisplay}
+              textColor={rightTitleColor}
               tag={TextTagType.DIV}
             >
               {rightTitle}
             </Text>
           )}
           {rightBody && (
-            <Text className="kirk-item-body" display={rightBodyDisplay} tag={TextTagType.DIV}>
+            <Text
+              className="kirk-item-body"
+              display={rightBodyDisplay}
+              textColor={rightBodyColor}
+              tag={TextTagType.DIV}
+            >
               {rightBody}
             </Text>
           )}
         </div>
       )}
       {rightAddon && <div className="kirk-item-rightAddon">{rightAddon}</div>}
-      {chevron && <div className="kirk-item-rightAddon">{<ChevronIcon />}</div>}
+      {chevron && (
+        <div className="kirk-item-rightAddon">
+          {<ChevronIcon iconColor={!isClickable ? color.fadedText : color.secondaryText} />}
```

Comment:
Are these colors compatible with the left/right colors? We may end up with a rainbow-like item with this

Reply:
The chevron can only be two colors, from what I've seen: `secondaryText` by default, and a lighter grey when the Item is disabled. Even when the text is blue (recommended choice for example), the chevron stays grey.

I agree that there are a lot of color variations on Items, but I'm just following the specs here =/